spring-integration-test-support/src/main/java/org/springframework/integration/test/util/TestUtils.java

```diff
@@ -85,6 +105,11 @@ else if (i == tokens.length - 1) {
 		return (T) value;
 	}
 
+	/**
+	 * Create a {@link TestApplicationContext} instance
+	 * supplied with the besic Spring Integration infrastructure.
```

Comment:
Yes, that was an unlucky attempt to point to the typo (should be `basic`). I had been hesitating over a choice of expressing form for a while and at last chose the worst possible variant :smiley: